src/Session.php

```diff
@@ -550,4 +555,72 @@ protected function save()
 
         return $this->getStore()->save($this->getId(), $values, $this->ttl);
     }
+
+    /**
+     * Put a key / value pair or array of key / value pairs in the session.
+     *
+     * @param  string|array  $key
+     * @param  mixed       $value
+     * @return void
+     */
+    public function put($key, $value = null)
+    {
+        if (! is_array($key)) {
+            $key = [$key => $value];
+        }
+
+        foreach ($key as $arrayKey => $arrayValue) {
+            $this->setValue($arrayKey, $arrayValue);
+        }
+    }
+
+
+    /**
+     * Push a value onto a session array.
+     *
+     * @param  string  $key
+     * @param  mixed   $value
+     * @return void
+     */
+    public function push($key, $value)
+    {
+        $array      = $this->getValue($key);
+        $array[]    = $value;
+        $this->setValue($key, $array);
+    }
+
+    /**
+     * Remove one or many items from the session.
+     *
+     * @param  string|array  $keys
+     * @return void
+     */
+    public function forget($keys)
+    {
+        $remove  = $keys;
+
+        foreach($remove as $key){
+            $this->unsetValue($key);
+        }
+
+    }
+
+
+    /**
+     * Call the flash session handler.
+     *
+     * @return SessionFlash
+     */
+    public function flash()
+    {
+        if (is_null($this->flash)) {
+
+            $this->flash = new SessionFlash($this);
+
+            return $this->flash;
```

Comment:
I think one `return $this->flash` is enough. Just put it outside curly braces.

Reply:
I do not understand.

Can you make an example?

Reply:
Sure.

```
    public function flash()
    {
        if (is_null($this->flash)) {
            $this->flash = new SessionFlash($this);
        }

        return $this->flash;
    }
```

It works exactly same way, but method is smaller.  It's all about setting default value when `flash` is empty and you want to use it.

Reply:
Ok!